### ISSUE TITLE
Add a "Return to start" button

### ISSUE THREAD

**Issue by cjonsmith** [DanceParty/MSHackGeoGuesser#50]

**Comment by DanceParty** [comment]
during the game? So you can return to the location you started at ?

**Comment by cjonsmith** [comment]
Yep; that way if you traveled far away from the start you can quickly return to the spot where you need to make your guess.

**Comment by DanceParty** [comment]
That feature's only for tryhards...

**Comment by kfcampbell** [comment]
I think this is blocked pending a couple of things:
- #31 and #32 are more important IMHO
- we need to make a decision about whether we score you from your current lat/long or where you started.

That is, if I land somewhere in France, can I "drive" to Paris and take the score there for perfect accuracy?

**Comment by DanceParty** [comment]
I think we should score from where we drop them imo
